test/orchestratord/mzcompose.py

```diff
@@ -1127,6 +1132,227 @@ class Action(Enum):
+            ]
+        )
+        spawn.runv(["helm", "repo", "update", "materialize"])
+        spawn.runv(
+            [
+                "helm",
+                "install",
+                "my-materialize-operator",
+                MZ_ROOT / "misc" / "helm-charts" / "operator",
+                "--namespace=materialize",
+                "--create-namespace",
+                "--version",
+                "v25.3.0",
+                "--set",
+                "observability.podMetrics.enabled=true",
+                "-f",
+                os.path.join(dir, "sample-values.yaml"),
+            ]
+        )
+        spawn.runv(
+            ["kubectl", "apply", "-f", os.path.join(dir, "sample-postgres.yaml")]
+        )
+        spawn.runv(["kubectl", "apply", "-f", os.path.join(dir, "sample-minio.yaml")])
+        spawn.runv(["kubectl", "get", "all", "-n", "materialize"])
+        spawn.runv(
+            [
+                "helm",
+                "repo",
+                "add",
+                "metrics-server",
+                "https://kubernetes-sigs.github.io/metrics-server/",
+            ]
+        )
+        spawn.runv(["helm", "repo", "update", "metrics-server"])
+        spawn.runv(
+            [
+                "helm",
+                "install",
+                "metrics-server",
+                "metrics-server/metrics-server",
+                "--namespace",
+                "kube-system",
+                "--set",
+                "args={--kubelet-insecure-tls,--kubelet-preferred-address-types=InternalIP,Hostname,ExternalIP}",
+            ]
+        )
+        for i in range(120):
+            try:
+                spawn.capture(
+                    [
+                        "kubectl",
+                        "get",
+                        "crd",
+                        "materializes.materialize.cloud",
+                        "-n",
+                        "materialize",
+                        "-o",
+                        "name",
+                    ],
+                    stderr=subprocess.DEVNULL,
+                )
+                break
+
+            except subprocess.CalledProcessError:
+                pass
+            time.sleep(1)
+        else:
+            raise ValueError("Never completed")
+
+        with open(os.path.join(dir, "sample-materialize.yaml")) as f:
+            materialize_setup = list(yaml.load_all(f, Loader=yaml.Loader))
+        assert len(materialize_setup) == 3
+
+        if version == current_version:
+            materialize_setup[2]["spec"]["environmentdImageRef"] = get_image(
+                c.compose["services"]["environmentd"]["image"], args.tag
+            )
+            # Self-managed v25.1/2 don't require a license key yet
+            materialize_setup[1]["stringData"]["license_key"] = os.environ[
+                "MZ_CI_LICENSE_KEY"
+            ]
+        else:
+            # TODO: Remove this part once environmentId is set in older versions
```

Comment:
Can we just set this in our provided sample yaml file to make life easier for customers?

Reply:
Shouldn't it be unique per customer, though?

Reply:
Then we should at least document a simple way to set it, ideally with a script that generates a uuid for them and puts it in automatically.